crates/web-sys/Cargo.toml

```diff
@@ -632,7 +634,7 @@ IirFilterOptions = []
 ImageBitmap = []
 ImageBitmapFormat = []
 ImageBitmapRenderingContext = []
-ImageCapture = ["EventTarget"]
+ImageCapture = []
```

Comment:
(this wasn't related to the WebGPU changes, but it was removed when I regenerated the bindings)